File: src/haskell/app/Const.hs
module Const where

columns :: Int
columns = 100
rows :: Int
rows = 100

padding :: Int
padding = 100

cellDimenstions :: Int
cellDimenstions = 10
cellDimensionsFloat :: Float
cellDimensionsFloat = fromIntegral cellDimenstions

gridHeight :: Float
gridHeight = fromIntegral rows * cellDimensionsFloat

gridWidth :: Float
gridWidth = fromIntegral columns * cellDimensionsFloat

windowHeight :: Int
windowHeight = rows * cellDimenstions + padding

windowWidth :: Int
windowWidth = columns * cellDimenstions + padding

File: src/go/const.go
package main

const (
	columns        int = 100
	rows           int = 100
	padding        int = 50
	cellDimensions int = 10
)

var (
	cellDimensionsFloat float32 = float32(cellDimensions)
	gridHeight          float32 = float32(rows) * cellDimensionsFloat
	gridWidth           float32 = float32(columns) * cellDimensionsFloat
	windowHeight        int     = rows*cellDimensions + padding*2
	windowWidth         int     = columns*cellDimensions + padding*2
)
